users/mechmerlin/rules.mk

```diff
@@ -1 +1,6 @@
-SRC += mechmerlin.c
\ No newline at end of file
+SRC += mechmerlin.c
+
+ifdef RGB_DI_PIN
```

Comment:
This is a way better method of enabling/disabling features than checking the keyboard name at compile time. I'm gonna need to make a mental note to steal this bit of code in the future. :+1:

Reply:
=)

Reply:
This won't work though. 

It's not defined nor imported into the makefiles, as far as I'm aware. 

You'd be better off checking for a specific keyboard.

Reply:
... buzzkill.

Reply:
I wish it was that easy.  

You could use this, though
```suggestion
ifeq ($(strip $(RGBLIGHT_ENABLE)), yes)
```